01week/bio/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Samantha's Bio Page</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
```

Comment:
to link your bio stylesheet to this html, I believe you need to change the "style.css" to "css/style.css"